### PR TITLE
Fix extra scroll bars

### PR DESCRIPTION
**Pull request by Toxiapo**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1371)

### This PR does the following:
- Upgrade webreader to 0.3.3
- CSS update to fix vertical scroll bar 

### Testing requirements & instructions: 
-

### CODE REVIEW

File: styles/main.scss [hunk 1]
```diff
@@ -41,6 +41,12 @@ $fontFamily: "system-ui";
 
 @import "./dividers.scss";
 
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
```
Comment (Toxiapo) [inline styles/main.scss:49]: It looks like I still need this line : ( The vertical scrollbar only shows up on a big monitor.

File: styles/main.scss [hunk 2]
```diff
@@ -50,7 +56,6 @@ body {
 }
 
 .nypl-ds {
-  overflow: unset;
```
Comment (Toxiapo) [inline styles/main.scss:53]: Do we need this? I think we do need the overflow to be hidden(default) or it shows a horizontal bar on the landing page.

**Comment by hokei** [comment]
@mwbenowitz can we figure out why this didn't trigger a test deploy?  Is that functionality that got reverted somehow and we missed it?

**Comment by Toxiapo** [comment]
This is the preview link from the dashboard, I think we just need to fix the auto-share feature. (I don't have the admin to see the settings, it takes me to a broken page after I click on the settings button)

https://pr305-aic8bhujmbqyp9sxs67v9lhj3oqevhne.tugboat.qa/

File: styles/main.scss (hunk 2 above)
Comment (hokei) [reply styles/main.scss:53]: I think it's ok for now.  This definitely will be a thing we're changing soon with the DS upgrade

**Review by hokei — APPROVED** [review]
This looks great!  I checked the Tugboat settings and fiddled with some things - we probably can't check it until a new PR gets built.  So, let's merge this in and see if the tugboat's fixed when we prepare this for release?

**Pull request merged**